ios/FluentUI/Extensions/UITableViewCell+Extension.swift

```diff
@@ -5,7 +5,7 @@
 
 import UIKit
 
-extension UITableViewCell {
+public extension UITableViewCell {
     func hideSystemSeparator() {
         separatorInset = UIEdgeInsets(top: 0, left: max(UIScreen.main.bounds.width, UIScreen.main.bounds.height), bottom: 0, right: 0)
```

Comment:
UIScreen usage is not encouraged (https://github.com/microsoft/fluentui-apple/issues/77)